index.js

```diff
@@ -104,11 +104,7 @@ async function installMicromambaPosix (micromambaUrl) {
       linux: 'wget -qO- --retry-connrefused --waitretry=10 -t 5'
     }[MAMBA_PLATFORM]
     const downloadCmd = `${downloadProg} ${micromambaUrl} | tar -xvjO bin/micromamba > ${PATHS.micromambaExe}`
-    try {
```

Comment:
It would be nice to have a retry of the micromamba download for a couple of times because there can always be some downtime (e.g. on the micromamba server or the anaconda server).

Reply:
Added something back.